tests/components/zeroconf/test_init.py

```diff
@@ -75,7 +74,7 @@ async def test_setup(hass, mock_zeroconf):
     expected_flow_calls = 0
     for matching_components in zc_gen.ZEROCONF.values():
         expected_flow_calls += len(matching_components)
-    assert len(mock_config_flow.mock_calls) == expected_flow_calls * 2
+    assert len(mock_config_flow.mock_calls) == expected_flow_calls
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5418277)